controllers/tests/config/selfnoderemediationconfig_controller_test.go

```diff
@@ -39,18 +44,43 @@ var _ = Describe("SNR Config Test", func() {
 
 	})
 
-	Context("DS installation", func() {
-		key := types.NamespacedName{
-			Namespace: shared.Namespace,
-			Name:      dsName,
+	AfterEach(func() {
+		if isSkipCleanup {
+			isSkipCleanup = false
+			return
 		}
 
+		tmpConfig := &selfnoderemediationv1alpha1.SelfNodeRemediationConfig{}
+		tmpDs := &appsv1.DaemonSet{}
+		//verify config exist
+		Eventually(func(g Gomega) {
+			g.Expect(k8sClient.Get(context.Background(), client.ObjectKeyFromObject(config), tmpConfig)).To(Succeed())
+		}, 10*time.Second, 250*time.Millisecond).Should(Succeed())
+
+		//ds may take quite a while to create (for example with a 5 sec timeout this test usually fails locally)
```

Comment:
This doesn't sound right, it should not need so long... maybe this is hiding another real issue in test setup or even production code 🤔 Will have a look...

Reply:
I run the test successfully over 10 times now, with 1 sec only here, and above... 

It fails on your laptop? Any chance your laptop is very busy with other stuff? Otherwise needing more than 5 seconds here doesn't make sense IMHO. However, since this doesn't slow down tests if they run fast, lgtm

/lgtm

Reply:
>It fails on your laptop?

Yep

>Any chance your laptop is very busy with other stuff?

I've retested and verified. CPU and Memory seems fine (I haven't check other criteria) 

In any case since we assume it's a resource associated issue I think I also prefer the "better safe than sorry" approach, especially considering previous experience with resource load in the CI environment.